client/landing/jetpack-cloud/components/masterbar/index.jsx

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Item from 'layout/masterbar/item';
+import JetpackLogo from 'components/jetpack-logo';
+import Masterbar from 'layout/masterbar/masterbar';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+export default function() {
+	return (
+		<Masterbar>
+			<Item
+				tipTarget="jetpack"
+				className="masterbar__item-home"
+				url="/"
+				tooltip="Jetpack Cloud Homepage"
```

Comment:
Just a note: This should be translated but not at this moment.

Reply:
Definitely. I added a todo comment in d02df7b so that it's on our radar.